web/client-api/src/main/java/io/deephaven/web/client/ide/IdeConnection.java

```diff
@@ -22,6 +22,9 @@
  */
 @JsType(namespace = "dh")
 public class IdeConnection extends QueryConnectable<IdeConnection> {
+    public static final String EVENT_DISCONNECT = "disconnect";
+    public static final String EVENT_RECONNECT = "reconnect";
```

Comment:
I don't see these being used - are these meant to be referenced by client code?

Reply:
Yes, we declare constants for events that can be fired on the class that can fire them.

Reply:
(sorry for the dup, i had this open and it didnt show the old message, so thought i hadnt responded...)